spec/javascript/components/lease_ups/__snapshots__/LeaseUpApplicationsPage.test.js.snap

```diff
@@ -427,7 +427,7 @@ exports[`LeaseUpApplicationsPage Should render LeaseUpTable 1`] = `
                                           id="form-preference"
                                         />
                                         <select
-                                          className="form-group"
+                                          className=""
```

Comment:
Same className removal here

Reply:
this is just the snapshot, I tried setting it to null or undefined and it still shows up as a line